modules/core/src/main/java/org/apache/ignite/internal/lang/IgniteInternalCheckedException.java

```diff
@@ -131,7 +136,7 @@ public IgniteInternalCheckedException(UUID traceId, int code, Throwable cause) {
      * @param message Detail message.
      * @param cause Optional nested exception (can be {@code null}).
      */
-    public IgniteInternalCheckedException(int code, String message, Throwable cause) {
+    public IgniteInternalCheckedException(int code, String message, @Nullable Throwable cause) {
```

Comment:
Maybe we can correct the description of the constructor for this field then?

Reply:
What's wrong with the current description?